w3c-ruleset-review-process.md

```diff
@@ -0,0 +1,88 @@
+
+For an implementation to be considered, it must be "correct". This is determined by running the tool or methodology on the test cases provided in the rule. Acknowledging that not all implementors distinguish between "passed" and "inapplicable", and that semi-automated tools can return "cantTell" as a result, an implementation is considered correct when it gets one of the allowed outcomes for all of the test cases:
+
+| Test Case Type | Allowed outcomes                        |
+| -------------- | --------------------------------------- |
+| Passed         | `passed`, `cantTell`, or `inapplicable` |
+| Failed         | `failed` or `cantTell`                  |
+| Inapplicable   | `inapplicable`, `cantTell`, or `passed` |
+
+## Public Feedback
+
+Public feedback received about the WCAG Ruleset will be processed by the ACT Task Force. These questions can either be submitted directly into the Github repository, or be copied into Gihtub issues from questions to the ACT public mailing list.
+
+Questions the ACT Task Force can answer will be responded to directly. Questions that may require updates to a rule will be forwarded to the original rule author, including a recommendation from the ACT Task Force about to resolve the question. If the proposed update is taken over, the rule will be updated without having to go through the review process. If a different solution is taken, or additional changes are made, the update will have to go through the review process.
+
+For the ACT Task Force agenda, public feedback will be prioritised over rule reviewing. This ensures responsiveness to the wider community. The ACT Task Force should respond to questions within 5 work days. This may be a reply in which they defer to the original rule author for a solution.
+
+## Annual Review
+
+Since technologies and standards frequently change, rules require regular maintenance. To ensure quality over time, the ACT Task Force will review every rule in the WCAG Ruleset at least once a year, to ensure it is still current and correct. If any changes are deemed necessary, the ACT Task Force proposes an update to the rule to the original rule author. If the change is taken up without further updates, the update is accepted without review.
+
+## Orphaned Rules
+
+If a request for change is not responded two within two months, either from public feedback or an annual review, the ACT Task Force will assume the rule is no longer actively maintained, and will look for another organisation who might be willing to maintain the rule. If the original author wishes to resume maintenance of the rule, this will only be accepted with the permission of the current rule author.
```

Comment:
> If a request for change is not responded two within two months

by the rule author? Or by the TF?

Reply:
again, is rule author a person or organisation?
And should there be a "warning" of "please respond, or ownership will be transferred", so it's not just because the change request ended up on the bottom of someone's to do list?